nginx/nginx.go

```diff
@@ -202,6 +231,23 @@ func (lb *nginxLoadBalancer) periodicallyUpdateMetrics() {
 	}
 }
 
+func (lb *nginxLoadBalancer) signaler() {
```

Comment:
Can we improve the function name, signaller is the thing that signals nginx and we already use that name. This is the loop.